packages/boilerplate-generator/boilerplate_web_browser_template.js

```diff
@@ -0,0 +1,70 @@
+// Template function for rendering the boilerplate html for browsers
+// Replicates the template defined in boilerplate_web.browser.html
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+export default function (manifest) {
+  const root = manifest;
+  // XXX do we need to do some validation on the properties of root?
+  return [].concat(
+    [
+      '<html' +_.map(root.htmlAttributes, (value, key) =>
+        _.template(' <%= attrName %>="<%- attrValue %>"')({
+          attrName: key,
+          attrValue: value
+        })
+      ).join('') + '>',
+      '<head>'
+    ],
+
+    _.map(root.css, ({url}) =>
+      _.template('  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="<%- href %>">')({
+        href: root.bundledJsCssUrlRewriteHook(url)
+      })
+    ),
+
+    [
+      root.head,
+      root.dynamicHead,
+      '</head>',
+      '<body>',
+      root.body,
+      root.dynamicBody,
+      '',
+      _.template(root.inlineScriptsAllowed
+        ? '  <script type="text/javascript"><%= contents %></script>'
+        : '  <script type="text/javascript" src="<%- src %>"></script>'
+      )({
+        contents: _.template('__meteor_runtime_config__ = JSON.parse(decodeURIComponent(<%= conf %>))')({
```

Comment:
Not sure why you're doing nested calls to _.template instead of just putting this in the first template. Also, it's kinda weird that you're providing both keys (contents/src) to either template... why not do

```
root.inlineScriptsAllowed ?
  _.template('template1', args1) : _.template('template2', args2)
```
?

Also in the old version, `meteorRuntimeConfig` was `{{}}` (HTML-escaped) though it probably doesn't matter due to the use of encodeURIComponent.